src/model/game/GameManager.ts

```diff
@@ -364,6 +364,36 @@ export default class GameManager {
             "https://thunderstore.io/c/ancient-dungeon-vr/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
             [new StorePlatformMetadata(StorePlatform.STEAM, "1125240"), new StorePlatformMetadata(StorePlatform.OCULUS_STORE)], "ancient-dungeon-vr.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.ANCIENT_DUNGEON_VR, ["adv"]),
+
+        new Game("RUMBLE", "RUMBLE", "RUMBLE",
+            "RUMBLE", ["RUMBLE.exe"], "RUMBLE_Data",
+            "https://thunderstore.io/c/rumble/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "890550")], "RUMBLE.png",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.MELON_LOADER, []),
+
+        new Game("Dome Keeper", "DomeKeeper", "DomeKeeper",
+            "Dome Keeper", ["domekeeper.exe"], "",
+            "https://thunderstore.io/c/dome-keeper/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "1637320")], "dome-keeper.jpg",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.GODOT_ML, []),
+
+        new Game("Skul: The Hero Slayer", "SkulTheHeroSlayer", "SkulTheHeroSlayer",
+            "Skul", ["Skul.exe"], "Skul_Data",
+            "https://thunderstore.io/c/skul-the-hero-slayer/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
+            [new StorePlatformMetadata(StorePlatform.STEAM, "1147560")], "skul-the-hero-slayer.jpg",
+            GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, []),
```

Comment:
Likewise - "sths" is probably fine

Reply:
I thought about this but I'm pretty sure the actual shorthand for the game is just "skul", which is already discoverable